packages/config/config/devices/0x010f/fgpb101.json

```diff
@@ -53,179 +91,460 @@
 	"paramInformation": {
 		"1": {
 			"label": "Scenes sent to the controller",
-			"description": "This parameter determines which actions result in sending scene IDs to them.",
+			"description": "This parameter determines which actions result in sending scene IDs and attributes assigned to them.\n1 - Key Pressed 1 time\n2 - Key Pressed 2 times\n4 - Key Pressed 3 times\n8 - Key Pressed 4 times\n16 - Key Pressed 5 times\n32 - Key Held Down\n64 - Key Released\nNOTE\nValues of parameter 1 may be combined, e.g. 1+2=3 means that scenes will be sent \nafter pressing the button once or twice.",
 			"valueSize": 1,
 			"minValue": 1,
-			"maxValue": 127,
+			"maxValue": 64,
 			"defaultValue": 127,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Key Held Down",
+					"value": 32
+				},
+				{
+					"label": "Key Released",
+					"value": 64
+				},
+				{
+					"label": "Key Pressed 1 time",
+					"value": 1
+				},
+				{
+					"label": "Key Pressed 2 times",
+					"value": 2
+				},
+				{
+					"label": "Key Pressed 3 times",
+					"value": 4
+				},
+				{
+					"label": "Key Pressed 4 times",
+					"value": 8
+				},
+				{
+					"label": "Key Pressed 5 times",
+					"value": 16
+				}
+			]
```

Comment:
That's a bitmask. We need to convert this to partial params

Reply:
I went back to manual entry for now, unless you can tell me how it should look. I haven't had time to look into it yet.